content/en/tracing/metrics/metrics_namespace.md

```diff
@@ -39,21 +39,18 @@ Trace metrics are generated for service entry spans and certain operations depen
 The [trace metrics][3] namespace is formatted as:
 
 - `trace.<SPAN_NAME>.<METRIC_SUFFIX>`
-- `trace.<SPAN_NAME>.<METRIC_SUFFIX>.<2ND_PRIM_TAG>_service`
```

Comment:
There is no metric suffix for trace distribution metrics. Might be worth a call-out?

Reply:
yep it is a line I removed